docs/generated/http/defs.bzl

```diff
@@ -0,0 +1,63 @@
+# PROTOBUF_TARGETS lists the indirect dependencies needed to compile proto dependencies
+# of //pkg/server/serverpb:serverpb_proto target
+PROTOBUF_TARGETS = [
+    "//pkg/build:build_proto",
+    "//pkg/clusterversion:clusterversion_proto",
+    "//pkg/config/zonepb:zonepb_proto",
+    "//pkg/geo/geopb:geopb_proto",
+    "//pkg/gossip:gossip_proto",
+    "//pkg/jobs/jobspb:jobspb_proto",
+    "//pkg/kv/kvpb:kvpb_proto",
+    "//pkg/kv/kvserver/concurrency/isolation:isolation_proto",
+    "//pkg/kv/kvserver/concurrency/lock:lock_proto",
+    "//pkg/kv/kvserver/kvserverpb:kvserverpb_proto",
+    "//pkg/kv/kvserver/liveness/livenesspb:livenesspb_proto",
+    "//pkg/kv/kvserver/loqrecovery/loqrecoverypb:loqrecoverypb_proto",
+    "//pkg/kv/kvserver/readsummary/rspb:rspb_proto",
+    "//pkg/kv/kvserver/kvflowcontrol/kvflowcontrolpb:kvflowcontrolpb_proto",
```

Comment:
@rickystewart , FYI had to add `pkg/kv/kvserver/kvflowcontrol/kvflowcontrolpb:kvflowcontrolpb_proto` target as it is a transient dependency for `serverpb`.

Reply:
LGTM